pomo.sh

```diff
@@ -63,7 +63,7 @@ function pomo_stop {
 function pomo_ispaused {
     # Return 0 if paused, 1 otherwise.
     # pomo.sh is paused if the POMO file contains any information.
-    [[ $(wc -l "$POMO" | cut -d" " -f1) -gt 0 ]]
+    [[ $(wc -l "$POMO" | sed 's/^ *//' | cut -d" " -f1) -gt 0 ]]
```

Comment:
Just to make this test a bit cleaner -- does `[[ $(wc -l < "$POMO") -gt 0 ]]` also work, or does it have a similar problem with leading whitespace on your system?

Reply:
Interesting solution! It works on both versions -- GNU as well as BSD wc.

Reply:
Yeah, it exploits bash just does type conversion when necessary for operations (such as numeric comparison) and that passing wc input as STDIN means it doesn't have a filename to show.

If you update the commit to use that test instead (we also lose an unnecessary cut), then I'll merge it. Thanks again for spotting and fixing an incompatibility!